tests/cpu_bugs/reboot_and_wait_up.pm

```diff
@@ -0,0 +1,54 @@
+# SUSE's openQA tests
```

Comment:
I think we have a very similar file if not already the same multiple times, please don't add it.

Reply:
Yes, I will review it to see whether I could integrate it into lib/Mitigation.pm. or drop it.